slick/src/main/scala/slick/jdbc/DerbyProfile.scala

```diff
@@ -240,6 +243,27 @@ trait DerbyProfile extends JdbcProfile {
       override def sqlType = java.sql.Types.BINARY
       override def sqlTypeName(sym: Option[FieldSymbol]) = "CHAR(16) FOR BIT DATA"
     }
+
+    class InstantJdbcType extends super.InstantJdbcType {
+      // Derby has no timestamp with timezone type and so using strings as timestamps are
+      // susceptible to DST gaps twice a year
```

Comment:
this comment should not apply to Instant, which should be stored in Z-time, i.e. without DST